integration-test-processor/src/main/java/org/springframework/cloud/stream/module/it/IntegrationTestProcessorProperties.java

```diff
@@ -56,6 +57,12 @@
 	 */
 	private String parameterThatMayNeedEscaping;
 
+	/*
+	 * Same as ChannelBindingServiceProperties.instanceIndex.

```

Comment:
Because this now mimics what is handled by `ChannelBindingServiceProperties.instanceIndex`. This is intended for a test that deploys multiple instances of the same app. But I realize that test never made it to `AbstractAppDeployerIntegrationTests` o.O ?